src/golang/lib/engine/aq_engine.go

```diff
@@ -347,6 +362,20 @@ func (eng *aqEngine) PreviewWorkflow(
 		return nil, errors.Wrap(err, "Unable to initialize vault.")
 	}
 
+	var jobManager job.JobManager
+	if dbDAG.EngineConfig.Type == shared.DatabricksEngineType {
+		jobManager, err = job.GenerateNewJobManager(
+			ctx,
+			dbDAG.EngineConfig,
+			&dbDAG.StorageConfig,
+			eng.AqPath,
+			vaultObject,
+		)
+		if err != nil {
+			return nil, errors.Wrap(err, "Unable to generate a new JobManager.")
```

Comment:
I think this errors wrapping is unnecessary, the returned error already has the appropriate context.